ami/jobs/management/commands/test_ml_job_e2e.py

```
@@ -0,0 +1,127 @@
+
+        # Single line output with carriage return
+        progress_line = " | ".join(progress_parts)
+        self.stdout.write(f"\r{progress_line:<100}", ending="")
+
+    def _show_final_results(self, job, start_time):
+        total_time = time.time() - start_time
+
+        self.stdout.write(f"\n\n{'='*50}")
+
+        if job.status == JobState.SUCCESS:
+            self.stdout.write(self.style.SUCCESS(f"✅ Job completed successfully"))
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)